vts/trustedservices/trustedservices_grpc.go

```diff
@@ -418,17 +424,22 @@ func (c *GRPC) initEvidenceContext(
 	return appraisal, err
 }
 
-func (c *GRPC) getTrustAnchor(id string) (string, error) {
-	values, err := c.TaStore.Get(id)
-	if err != nil {
-		return "", err
-	}
+func (c *GRPC) getTrustAnchors(id []string) ([]string, error) {
+
+	ta_values := make([]string, 0)
+	for _, taId := range id {
+		values, err := c.TaStore.Get(taId)
+		if err != nil {
+			return []string{""}, err
+		}
 
-	if len(values) != 1 {
-		return "", fmt.Errorf("found %d trust anchors, want 1", len(values))
+		if len(values) != 1 {
+			return []string{""}, fmt.Errorf("found %d trust anchors, want 1", len(values))
+		}
```

Comment:
This is an interesting restriction.  Do we want to remove it?  Or leave it in but document why we do it?

Reply:
Added a comment to state the current restriction. We can revisit this if we want to support multiple TA's per 
TAID. Right now none of the schemes have this support!

Reply:
> Added a comment to state the current restriction. We can revisit this if we want to support multiple TA's per TAID. Right now none of the schemes have this requirement !